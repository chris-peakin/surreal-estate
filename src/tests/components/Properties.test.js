import { render } from "@testing-library/react";
import Properties from "../../components/PropertiesComponent";

test("properties page renders", () => {
  const { asFragment } = render(<Properties />);
  expect(asFragment()).toMatchSnapshot;
});
