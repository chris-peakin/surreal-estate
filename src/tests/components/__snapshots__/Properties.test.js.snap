// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`properties page renders 1`] = `
<DocumentFragment>
  <div />
  <div
    class="individual_properties"
  />
</DocumentFragment>
`;
